NomadPlat/src/Linux/PlatformWindowLinux.cpp

```
@@ -0,0 +1,307 @@
+}
+
+void PlatformWindowLinux::setCursorVisible(bool visible) {
+    SDL_ShowCursor(visible ? SDL_ENABLE : SDL_DISABLE);
+}
+
+KeyModifiers PlatformWindowLinux::getCurrentModifiers() const {
+    return getModifiers(SDL_GetModState());
+}
+
+// Helpers
+KeyCode PlatformWindowLinux::translateKey(SDL_Keycode key) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)